<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDOPCUAServerParaForm</class>
 <widget class="QWidget" name="CMDOPCUAServerParaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="spTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>219</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>动作</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>安全策略</string>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="spinBox_port">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>4840</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_auth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>授权</string>
   </property>
   <widget class="QRadioButton" name="rb_anonymous">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>匿名</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_userPassword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>账户密码</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_authFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>认证文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cert">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>证书:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>私钥:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_pkey">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
